src/main/resources/schemas/single_justice_procedure_public.json

```diff
@@ -11,7 +11,7 @@
           "examples": [
             "PR1 2LL"
```

Comment:
Can we update the example here too to reflect the change

Reply:
Example updated